packages/core/src/sass/app/_common/layout/app-layouts.scss

```
@@ -69,6 +69,16 @@
             position: absolute;
             right: 1.6rem;
         }
+        .dc-toast-popup-mobile {
+            display: inherit;
+            background-color: var(--general-main-1);
+            max-width: calc(100vw - 1.6rem);
+            left: 1.2rem;
+            width: calc(100vw - 1.2rem);
+            top: 0.4rem;
+            position: absolute;
+            right: 1.2rem;
```

best to group the position css together
```suggestion
            position: absolute;
            width: calc(100vw - 1.2rem);
            top: 0.4rem;
            left: 1.2rem;
            right: 1.2rem;
```